content/docs/start/data-pipelines.md

```diff
@@ -247,7 +247,6 @@ Now, let's change it back to `50` and run `dvc repro` again:
 $ dvc repro
 Stage 'prepare' didn't change, skipping
 Stage 'featurize' didn't change, skipping
-Restored stage 'train' from run-cache
```

Comment:
I assume this one was unintended (prob. don't want to confuse beginners with a run-cache mention here).